lua/starfall/libs_cl/midi.lua

```diff
@@ -0,0 +1,82 @@
+if not SF.Require("midi") then return function() end end
+
+--Midi Library
+-- @name midi
+-- @class library
+-- @libtbl midi_library
+SF.RegisterLibrary("midi")
+
+return function(instance)
+-- Only usable by the owner of the starfall chip
+if LocalPlayer() ~= instance.player then return end
+
+local midi_library = instance.Libraries.midi
+
+-- Event hook for midi
```

Comment:
Probably want to explain more about when this will trigger

Reply:
done